### PR TITLE
Resolved grafana comments

### PR DESCRIPTION
**Pull request by diericd**

### CODE REVIEW

File: docs/content/1_release_notes/_index.md [hunk 1]
```diff
@@ -21,6 +21,14 @@ This is the compatibility matrix for version listed on this page. If you have do
...
+
+### Changes
+
+- Pass datasource request context to the historian client
+- Automatic instance management for data source
+- Removed pprof code
+
 ## v2.0.0
 
 Released on IDK
```
Comment (vereecw) [inline docs/content/1_release_notes/_index.md:34]: Remove this

File: pkg/datasource/historian.go [hunk 2]
```diff
@@ -17,47 +24,29 @@ const (
...
 	if err != nil {
 		return nil, err
 	}
 
-	return historianPlugin, nil
+	return historianDataSource, nil
 }
+
+// Dispose here tells plugin SDK that plugin wants to clean up resources when a new instance
+// created.
```
Comment (vereecw) [inline pkg/datasource/historian.go:51]:
```suggestion
// Dispose here tells plugin SDK that plugin wants to clean up resources when a new instance
// is created.
```

File: pkg/datasource/historian.go [hunk 3]
```diff
@@ -17,47 +24,29 @@ const (
 
 // HistorianPlugin ...
 type HistorianPlugin struct {
-	API *api.API
 }
```
Comment (vereecw) [inline pkg/datasource/historian.go:27]: Can be removed

**Review by vereecw — APPROVED** [review]

File: pkg/main.go [hunk 4]
```diff
@@ -4,15 +4,13 @@ import (
...
 	"github.com/grafana/grafana-plugin-sdk-go/backend/datasource"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
 )
 
 func main() {
-	backend.SetupPluginEnvironment(historianDataSource.PluginID)
-	err := datasource.Serve(historianDataSource.NewDataSource())
-	if err != nil {
-		backend.Logger.Error("error loading plugin", "pluginId", historianDataSource.PluginID)
+	if err := datasource.Manage("factry-historian-datasource", historianDataSource.NewDataSource, datasource.ManageOpts{}); err != nil {
```
Comment (vdpoora) [inline pkg/main.go:12]: use historianDataSource.PluginID

**Pull request merged**